ReactAndroid/src/androidTest/assets/AndroidTestBundle.js.meta

```diff
@@ -0,0 +1 @@
+5	 4f:}&�		dl�8mH
```

Comment:
@satya164 While running test cases that file get created. I removed that file and updated the pull request.